test/functionalTest/cases/1259_new_json_for_v2/update_attribute_with_null_value.test

```diff
@@ -0,0 +1,534 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Attribute With NULL Value, only APIv2
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB 0
+
+--SHELL--
+
+#
+# 01. Create (POST /v2/entities) E1/A1=null
+# 02. Get E1/A1
+# 03. Create E2/A2='03'
+# 04. Modify E2/A2=null (PATCH)
+# 05. Get E2/A2 (A2 stays the same)
+# 06. Create E3/A3='06'
+# 07. Modify E3/A3=null (PUT)
+# 08. Get E3/A3 (value of A3 has been replaced)
+# 09. Create E4/A4='09'
+# 10. Modify E4/A4=null (POST)
+# 11. Get E4/A4
```

Comment:
Expectations for this GET should be explained between brakets (same way than in step 5 and 8).

Same for 13, 16 and 19.